Hudl.Mjolnir/Command/Command.cs

```diff
@@ -343,10 +360,16 @@ private Task<TResult> ExecuteInIsolation(CancellationToken cancellationToken)
 
             var workItem = ThreadPool.Enqueue(() =>
             {
+               
+                if (TimeoutsIgnored)
+                {
+                    return UseCircuitBreakers.Value
+                        ? ExecuteWithBreaker(CancellationToken.None)
+                        : ExecuteAsync(CancellationToken.None);
+                }
```

Comment:
yep, valid point. I think I refactored this multiple times and so ended up with less than neat code here.